jscomp/syntax/src/res_core.ml

```diff
@@ -1397,6 +1400,29 @@ and parse_list_pattern ~start_pos ~attrs p =
     let pat = make_list_pattern loc patterns None in
     {pat with ppat_loc = loc; ppat_attributes = attrs}
 
+and parse_dict_pattern_row p =
```

Comment:
These 2 functions look good. I guess that already minimise the code duplication wrt normal records as they are very short.